src/script/user/UserMapper.ts

```diff
@@ -119,7 +117,10 @@ export class UserMapper {
       userEntity.accent_id(accentId);
     }
 
-    userEntity.availability(availability);
+    if (availability !== undefined) {
+      // Availability should only change when it's a valid value (undefined should reset the availability)
```

Comment:
(undefined should reset the availability) -> (undefined **would** reset the availability)

Reply:
Ahaha I forgot a `not` in the sentence 😆